app/controllers/AuditController.scala

```diff
@@ -100,7 +101,7 @@ class AuditController @Inject() (implicit val env: Environment[User, SessionAuth
             val cityStr: String = Play.configuration.getString("city-id").get
             val tutorialStreetId: Int = Play.configuration.getInt("city-params.tutorial-street-edge-id." + cityStr).get
             val cityShortName: String = Play.configuration.getString("city-params.city-short-name." + cityStr).get
-            Future.successful(Ok(views.html.audit("Project Sidewalk - Audit", task, mission, region.get, Some(user), cityShortName, tutorialStreetId)))
+            Future.successful(Ok(views.html.audit("Project Sidewalk - Audit", task, Some(mission), region.get, Some(user), cityShortName, tutorialStreetId)))
```

Comment:
I think it would be perfectly reasonable to make a new page; I decided to use the audit page mainly for visual consistency 😁. But agree it seems like this is cluttering the code too much.